dev-python/dill/dill-0.3.0.ebuild

```diff
@@ -20,7 +20,7 @@ python_test() {
 	for t in tests/test_*.py; do
 		ebegin "\t${t}"
 		"${EPYTHON}" "${t}"
-		eend || fail=1
+		eend ${?} || fail=1
```

Comment:
Why `${?}` here, when it's `$?` everywhere else?

Reply:
Because it's my ebuild, and I prefer using `${}` consistently.